CIP-time-bound-delegation/README.md

```diff
@@ -0,0 +1,283 @@
+
+Let $W = \sum_i B_i$ over all eligible pools (whole circulation). Typically $W \approx 1$.
+
+Then normalize:
+
+$$
+f_{\mathrm{full}} = R \cdot \frac{B}{W} \;\approx\; (1+a_{0}) \cdot B = 16{,}435.354078
+$$
+
+```math
+\begin{aligned}
+\mathrm{ROI}_{\mathrm{epoch,full}} &= \frac{f_{\mathrm{full}}}{x+y} \\
+&= \frac{16{,}435.354078}{36{,}000{,}000} \\
+&= 0.000456537613
+\end{aligned}
+```
+
+```math
+\begin{aligned}
+\mathrm{ROI}_{\mathrm{annual,full}} &= \left(1 + \mathrm{ROI}_{\mathrm{epoch,full}}\right)^{73} - 1 \\
+&= 0.033880957 \;\; (\approx 3.3881\%)
+\end{aligned}
+```
+
+The result is about **3.39% Annual ROI**. Roughly a 0.79% increase.
+
+**Note:** These calculations do not take into account any collected transaction fees, pool performance, or pool fees.
+
+These effects are also modeled in the open source tool available at [https://spo-incentives.vercel.app/](https://spo-incentives.vercel.app/). Setting the rewards formula radio button to “Current” represents the current rewards calculation behavior. Setting it to “Full” represents the rewards calculation after applying this CIP. It is also recommended to experiment with the “Staked Ratio” slider under both of those settings to see how it will affect rewards.
+
+- **Runaway lost‑stake mitigation efficacy**
+
+  - Before:
+
+    ![Runaway lost-stake before mitigation (Fig. 2)](Fig2.png)
+
+  - After:
+
+    ![Runaway lost-stake after mitigation (Fig. 3)](Fig3.png)
+
+**Note:** The baseline numbers in the previous two charts were rough estimates taken from CPS-0022. These estimates may be off by a wide margin, but the effects in preventing the compounding of lost ADA beyond their expiration date will apply generally regardless.
+
+## Path to Active
+
+### Acceptance Criteria
+
+- **CIP Editor Approval** – Cardano CIP Editors must confirm that the specification is complete, unambiguous, and internally consistent with existing CIPs.
+- **Consensus on initial parameter values** – An initial values of for the new protocol parameters `stakePoolDelegationLifetime` and `drepDelegationLifetime` must be agreed upon before hard-fork combinator (HFC) activation. The choice should consider operational viability, empirical analyses (e.g., RSS results), and community feedback.
+- **Endorsement by Technical Bodies** – The Cardano Parameter-Change Proposals (PCP) Committee and the Intersect Technical Steering Committee (TSC) should both recommend the proposal as technically sound and aligned with the protocol’s long-term roadmap.
+- **Stakeholder Concurrence** – A majority of stake pool operators (SPOs), ecosystem tooling maintainers, dReps, and other infrastructure providers must signal readiness to upgrade.
+- **Governance Ratification** – The on-chain Hard-Fork Governance Action must pass the requisite dRep and Constitutional Committee thresholds, establishing legal-constitutional legitimacy and stakeholder support for the change.
```

Comment:
It may also make sense to update the constitution, and specifically adding guardrails around these parameters